guides/common/modules/proc_configuring-the-active-directory-authentication-source-on-projectserver.adoc

```diff
@@ -0,0 +1,122 @@
+[id="configuring-the-active-directory-authentication-source-on-projectserver_{context}"]
+= Configuring the Active Directory authentication source on {ProjectServer}
+
+Enable Active Directory (AD) users to access {Project} by configuring the corresponding authentication provider on your {ProjectServer}.
+
+.Prerequisites
+* The base system of your {ProjectServer} must be joined to an Active Directory (AD) domain.
+To enable AD users to sign in with Kerberos single sign-on, join the system by using the System Security Services Daemon (SSSD) and Samba services.
++
+Install the following packages on {ProjectServer}:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# {project-package-install} krb5-workstation oddjob-mkhomedir oddjob realmd samba-winbind-clients samba-winbind samba-common-tools samba-winbind-krb5-locator sssd
+----
++
+Specify the required software when joining the AD domain:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# realm join _AD.EXAMPLE.COM_ --membership-software=samba --client-software=sssd
+----
++
+For more information on direct AD integration, see link:{RHELDocsBaseURL}9/html-single/integrating_rhel_systems_directly_with_windows_active_directory/index#connecting-rhel-systems-directly-to-ad-using-samba-winbind_integrating-rhel-systems-directly-with-active-directory[Connecting RHEL systems directly to AD using Samba Winbind].
+
+.Procedure
+. Define AD realm configuration in a location where {foreman-installer} expects it:
+.. Create a directory named `/etc/ipa/`:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# mkdir /etc/ipa
+----
++
+.. Create a file named `default.conf` in the `/etc/ipa/` directory:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# touch /etc/ipa/default.conf
+----
+.. Add the following lines to `/etc/ipa/default.conf` to configure the Kerberos realm for the AD domain:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+[global]
+realm = _AD.EXAMPLE.COM_
+----
+. Configure the Apache keytab for Kerberos connections:
+.. Create a file named `/etc/net-keytab.conf` to store configuration details for Samba:
```

Comment:
Same as above. A step to create an empty file seems superfluous if the next step is "edit the file". An editor can create new files. :-)

Reply:
Is there a reason why not /etc/samba/smb.conf? 
Isn't there a risk that someone creates smb.conf later and then things get mixed up?

Reply:
I merged the steps.

Reply:
About smb.conf: That's very good insight. I'll test the procedure with smb.conf to see if it works that way too.

Reply:
It works! The procedure now instructs users to store the configuration in the default /etc/samba/smb.conf and this file is then used to generate the keytab.